runners/org.osbuild.fedora30

```diff
@@ -36,10 +38,11 @@ def nsswitch():
 
 
 if __name__ == "__main__":
-    ldconfig()
-    sysusers()
-    tmpfiles()
-    nsswitch()
+    with osbuild.api.exception_handler():
+        ldconfig()
+        sysusers()
+        tmpfiles()
+        nsswitch()
 
     r = subprocess.run(sys.argv[1:], check=False)
     sys.exit(r.returncode)
```

Comment:
Any reason to exclude the `subprocess` part here?

Reply:
Actually, good point!